wiki/Tournaments/CFC/2018/en.md

```diff
@@ -0,0 +1,241 @@
+   - [senya - Magica \[SatelliteFrostMix\]](https://osu.ppy.sh/beatmapsets/91791/#fruits/248474)
+- Freemod
+   - [nano - Nevereverland \[Rain\]](https://osu.ppy.sh/beatmapsets/149570/#fruits/439454)
+   - [Roselia - LOUDER \[Crystal's Rain\]](https://osu.ppy.sh/beatmapsets/595580/#fruits/1270273)
+   - [Hyadain - Hyadain no Joujou Yuujou (TV Size) \[Saten's SELAMAT PAGI!\]](https://osu.ppy.sh/beatmapsets/32775/#fruits/112586)
+- Tiebreaker 
+  - [UNDERVEIL ALLSTARS - UNDERVEIL IS REAL!!! ...AND WE ARE UNDERVEIL!!! \[*Together as one, we stay determined!\]](https://osu.ppy.sh/beatmapsets/636920/#fruits/1351241)
+  
+### Group Stage 
+
+**[Download the mappack here! (81.3MB)](https://mega.nz/#!k00gQAQA!AN6ivOW-s0QZZIqVwmj1VMZ1lw-Th3V9W0hlhDrX_vM)**
+
+- Nomod
+  - [ClariS - Hitorigoto -TV MIX- \[Spec's Rain\]](https://osu.ppy.sh/beatmapsets/624544/#fruits/1316296)
+  - [An - Artcore Jinja \[Sauci's Rain\]](https://osu.ppy.sh/beatmapsets/215155/#fruits/506536)
+  - [AKI AKANE - Kando*Reduction  \[Collab Insane\]](https://osu.ppy.sh/beatmapsets/262403/#osu/598863)
+  - [M2U - Marigold (feat. Guriri) \[Asaiga's Insane\]](https://osu.ppy.sh/beatmapsets/595496/#osu/1382836)
+- Hidden
+  - [FELT - Goldrop \[Spec's Rain\]](https://osu.ppy.sh/beatmapsets/204927/#fruits/506395)
+  - [Cream puff - Mermaid girl \[Another\]](https://osu.ppy.sh/beatmapsets/715915/#osu/1512693)
+- Hard Rock
+  - [Camellia - Ultimate Ascension \[Salad\]](https://osu.ppy.sh/beatmapsets/520301/#fruits/1105922)
+  - [yuikonnu - Byoumei Koiwazurai \[Hard\]](https://osu.ppy.sh/beatmapsets/135358/#osu/355623)
+- Double Time
+  - [Goose house - 18sai \[Platter\]](https://osu.ppy.sh/beatmapsets/242856/#fruits/560407)
+  - [All Levels at Once - Changeling  \[Insane\]](https://osu.ppy.sh/beatmapsets/68169/#osu/197482)
+- Freemod
+  - [Mili - Utopiosphere -Platonism- \[Tyrell's Utopia\]](https://osu.ppy.sh/beatmapsets/522132/#fruits/1111110)
+  - [DWB feat. Fade - One Reason (TV Size) \[Insane\]](https://osu.ppy.sh/beatmapsets/100892/#osu/323797)
+- Tiebreaker 
+  - [Sakamoto Maaya - Okaerinasai (tomatomerde Remix) \[Ascendance's Reunion\]](https://osu.ppy.sh/beatmapsets/544468/#fruits/1161865)
+  
+## Rules
+
+### General Rules
+
+* The tournament will be held in 1v1 double-elimination format with traditional seeding.
+* The matches will be played using Score V1 and Head to Head settings.
+* A referee will be present in every match.
+* The tournament will be held over 5 weekends, joining the Discord server is mandatory at least for the duration of the tournament.
+* First week-end matches will be best of 7, second week-end matches best of 9, third week-end matches are best of 11, and fourth and fifth week-end matches will be best of 13. 
+
+### Match Rules
+
+1. The referee will create the room and invite the players in.
+1. Theplayers will roll to determine the order of picking warmups and maps.
+1. The higher roller will pick the first warmup map.
```

Comment:
Change `map` to `beatmap`

Reply:
thank you, updated everything. gotta love dumb mistakes